backend/src/bundles/users/user.service.ts

```diff
@@ -15,24 +18,38 @@ class UserService implements Service {
         this.userRepository = userRepository;
     }
 
-    public find(
+    public async find(
         payload: Record<string, unknown>,
-    ): Promise<UserEntity | undefined> {
-        return this.userRepository.find({ ...payload });
+    ): Promise<UserFindResponseDto> {
+        const user = await this.userRepository.find({ ...payload });
+
+        if (!user) {
+            throw new Error('plug'); //TODO: This is plug
```

Comment:
? 
not sure what it is

Reply:
This is necessary to fix return undefined, as in the other branch.
This is a temporary code.